com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -682,52 +681,52 @@ public void OnProcessShader(Shader shader, ShaderSnippetData snippetData, IList<
             if (urpAsset == null || compilerDataList == null || compilerDataList.Count == 0)
                 return;
 
-            m_stripTimer.Start();
-
-            InitializeLocalShaderKeywords(shader);
-
             int prevVariantCount = compilerDataList.Count;
-            var inputShaderVariantCount = compilerDataList.Count;
-            for (int i = 0; i < inputShaderVariantCount;)
+
+            double stripTimeMs = 0;
+            using (TimedScope.FromRef(ref stripTimeMs))
```

Comment:
Yea git made this change confusing, but seems like the main change is this new scope using (TimedScope.FromRef(ref stripTimeMs))